pr_testing/test_multiple_prs.sh

```diff
@@ -1029,7 +1029,9 @@ if [ "X$BUILD_OK" = Xtrue -a "$RUN_TESTS" = "true" ]; then
   if [ $(echo ${ENABLE_BOT_TESTS} | tr ',' ' ' | tr ' ' '\n' | grep '^PROFILING$' | wc -l) -gt 0 ] ; then
     if $PRODUCTION_RELEASE ; then
       DO_PROFILING=true
-      mark_commit_status_all_prs 'profiling' 'pending' -u "${BUILD_URL}" -d "Waiting for tests to start"
+      for wf in $PROFILING_WORKFLOWS;do
```

Comment:
@gartung , `PROFILINH_WORKFLOWS` are not define here. you need to get these from `$CMS_BOT_DIR/cmssw-pr-test-config` just like https://github.com/cms-sw/cms-bot/blob/master/pr_testing/run-pr-profiling.sh#L4

Reply:
Thanks for spotting that. I was wondering why no profiling jobs started.